cms/extensions/users-permissions/config/functions/bootstrap.js

```diff
@@ -14,6 +14,11 @@ const uuid = require('uuid/v4');
 
 const usersPermissionsActions = require('../../../../node_modules/strapi-plugin-users-permissions/config/users-permissions-actions');
 
+const userProvidedServices = process.env.VCAP_SERVICES['user-provided'] || [];
```

Comment:
This might need a `JSON.parse`

Reply:
Ah yep silly oversight 👍

Reply:
@taylor-work is this the right place to put this logic? I wonder if we should localize it with the database config, so we have one place to define a config for local development.

Reply:
https://github.com/GSA/ai-experience-sharing-platform/blob/main/cms/config/cloud-foundry-data.js maybe?